### DISCUSSION TITLE
What is the best way to manually add a "host network mode" container?

### DISCUSSION THREAD

**Discussion by roydbt** [nginx-proxy/nginx-proxy#2382] [Q&A] [answered]
I am running Homebridge in a  "network_mode: host" docker container (tried doing it with the default bridge network mode but it doesn't work). What is the best way to manually add that container to the reverse proxy? Do I have to manually add it every time the nginx container boots up?

**Comment by buchdag** [comment, marked as answer]
Hi.

You should not need to manually add containers running in host networking mode since version 1.4.0, [but you'll need to correctly set the `VIRTUAL_PORT` environment variable on those containers](https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#host-networking).
